packages/name-controller/src/constants.ts

```diff
@@ -0,0 +1,105 @@
+// Below constants are intentionally copied from transaction-controller to avoid package dependency
```

Comment:
That's one good option, but some of the Etherscan logic is also very similar so I wasn't sure if a `@metamask/etherscan` could also be beneficial.

At this stage, I just knew we didn't want to depend on `@metamask/transaction-controller` and wanted to avoid dependencies if possible until usage of this new controller has been stabilised.

We'd be happy to remove this duplication in future though in a more focused ticket and PR.

Reply:
Okay! No problem.